src/components/shared/Project/VolumeStatLine/VolumeStatLine.tsx

```diff
@@ -0,0 +1,68 @@
+import { Trans } from '@lingui/macro'
+import ETHAmount from 'components/shared/currency/ETHAmount'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+
+import { ThemeContext } from 'contexts/themeContext'
+import { Property } from 'csstype'
+import { BigNumber } from 'ethers/lib/ethers'
+import { useCurrencyConverter } from 'hooks/v1/CurrencyConverter'
+import { useContext, useMemo } from 'react'
+import { formatWad } from 'utils/formatNumber'
+
+import { textPrimary, textSecondary } from 'constants/styles/text'
+import { CurrencyName } from 'constants/currency'
+import StatLine from '../StatLine'
+
+export const VolumeStatLine = ({
+  totalVolume,
+  color,
+  convertTo,
+}: {
+  totalVolume: BigNumber | undefined
+  color: Property.Color
+  convertTo?: { currency: CurrencyName }
+}) => {
+  const { theme } = useContext(ThemeContext)
+  const secondaryTextStyle = textSecondary(theme)
+
+  // TODO-wraeth: This is a v1 hook - do we need to update?
```

Comment:
Don't think so, probs should just move it to a shared dir though

Reply:
cool cool - might do it in another PR (the move) and just remove these TODOs

Reply:
Yeah sweet sweet sounds good

Reply:
all done - lemme know if there is anything else you need from this PR